pkg/trace/go.mod

```diff
@@ -12,13 +13,13 @@ require (
 	github.com/golang/protobuf v1.5.2
 	github.com/google/gofuzz v1.2.0
 	github.com/pkg/errors v0.9.1
-	github.com/shirou/gopsutil/v3 v3.22.3
+	github.com/shirou/gopsutil/v3 v3.22.2
```

Comment:
Any reason for this or was it just by chance?

Reply:
Just by chance. No idea what happened here tbh.